File: left/pyramid.hs
{- Haskell code
made in an online compiler -}
import Control.Monad(forM_)

pyramid :: IO()
pyramid = do
   print "Enter height of the pyramid: "

   line <- getLine
   let height = (read line :: Int)
   
   forM_ [1..height] $ \i -> do
      putStrLn(replicate i '*')

File: left/Pascal/pyramid.pas
{Pascal code
made in an online compiler}
program Pyramid;
var
  height:integer;
  i:integer;
  j:integer;
begin
  writeln('Enter height of the pyramid: ');
  readln(height);
  for i:= 1 to height do
    begin
      for j:= 1 to i do write('*'); 
      writeln();
    end
end.
